src/test-project/src/test/java/com/redhat/devtools/intellij/commonUiTestLibrary/AbstractLibraryTestBase.java

```diff
@@ -30,19 +34,33 @@
 
     protected static void createNewProject(String projectName, String projectType) {
```

Comment:
I could imagine having this functionality in some very general test util class, as one would like to reuse it multiple time on different test cases.